app/webpack/reactjs/components/open_studios/open_studios_registration_window.tsx

```diff
@@ -20,7 +18,7 @@ interface OpenStudiosRegistrationWindowProps {
   onClose: OpenStudiosRegWindowCloseFn;
 }
 
-const OpenStudiosRegistrationWindow: FC<OpenStudiosRegistrationWindowProps> = ({
+export const OpenStudiosRegistrationWindow: FC<OpenStudiosRegistrationWindowProps> = ({
```

Comment:
pulled the window content into it's own file.  it looks like a git move but it's because of the extracting of part of that file into a new component and then renaming the rest of it to a new place (registration modal)